src/modules/mavlink/streams/RAW_RPM.hpp

```diff
@@ -49,7 +49,7 @@ class MavlinkStreamRawRpm : public MavlinkStream
 
 	unsigned get_size() override
 	{
-		return _rpm_sub.advertised() ? MAVLINK_MSG_ID_RAW_RPM + MAVLINK_NUM_NON_PAYLOAD_BYTES : 0;
```

Comment:
Nice find. I was hoping someday we could learn from the common patterns that evolved and push some of it back into the generated mavlink headers in a more consumable (harder to misuse) form.